src/openMVG/cameras/Camera_Pinhole.hpp

```diff
@@ -108,11 +108,13 @@ class Pinhole_Intrinsic : public IntrinsicBase
   // Data wrapper for non linear optimization (update from data)
   virtual bool updateFromParams(const std::vector<double> & params)
   {
-    if (params.size() == 3) {
-      *this = Pinhole_Intrinsic(_w, _h, params[0], params[1], params[2]);
+    if (params.size() == 3)
+    {
+      this->setK(params[0], params[1], params[2]);
```

Comment:
Code is different from OpenMVG master https://github.com/openMVG/openMVG/blob/master/src/openMVG/cameras/Camera_Pinhole.hpp#L215